lib/util/util.go

```diff
@@ -46,3 +43,22 @@ func GetCommandOutput(cmd ...string) string {
 	}
 	return strings.TrimSpace(string(output))
 }
+
+var inputReader = bufio.NewReader(os.Stdin)
+
+func GetUserInput() string {
+	text, err := inputReader.ReadString('\n')
+	if err != nil && err != io.EOF {
```

Comment:
I may be able to remove this. Added this in when I parsing the user input wrong anyway. Basically is the error is `io.EOF`, then text is all the bytes read before the EOF was reached and thus I just want to return that.

https://golang.org/pkg/bufio/#Reader.ReadString